File: JAVASCRIPT/Cap03/c03ex02.js
// c03ex02.js

const rls = require('readline-sync');

console.log('CALCULO DE AREA DE UM CIRCULO');
console.log();

let R = rls.questionFloat('Entre valor do raio: ');

let A = Math.PI * Math.pow(R, 2);

console.log('Resultado = ', A.toFixed(2));
console.log();

process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();

File: CSHARP/Cap03/c03ex02.cs
// c03ex02.cs

using System;
using static System.Console;

public class Program
{
    public static void Main(string[] args)
    {
        double R, A;

        Write("Entre valor do raio: ");
        R = double.Parse(ReadLine());

        A = Math.PI * Math.Pow(R, 2);

        WriteLine("Resultado = {0:F2}", A);

        Write("\nAperte <Enter> para encerrar... ");
        ReadLine();
    }
}
